src/applications/my-education-benefits/components/GetFormHelp.jsx

```diff
@@ -1,22 +1,51 @@
 import React from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import { toggleValues } from 'platform/site-wide/feature-toggles/selectors';
+import FEATURE_FLAG_NAMES from 'platform/utilities/feature-toggles/featureFlagNames';
 
-function GetFormHelp() {
+function GetFormHelp({ showMebDgi40Features }) {
   return (
-    <>
-      <p className="help-talk">
-        If you need help with your application or have questions about
-        enrollment or eligibility, call our Education Call Center at{' '}
-        <a href="tel:888-442-4551">888-442-4551 (711)</a>. We're here Monday
-        through Friday, 8:00 a.m. to 7:00 p.m. ET. If you're outside the U.S.,
-        call us at <a href="tel:001-918-781-5678">001-918-781-5678</a>.
-      </p>
-      <p>
-        If you have technical difficulties using this online application, call
-        our MyVA411 main information line at{' '}
-        <a href="tel:800-698-2411">800-698-2411 (711)</a>. We're here 24/7.
-      </p>
-    </>
+    <p className="help-talk">
+      {showMebDgi40Features ? (
+        <>
+          If you need help with your application or have questions about
+          enrollment or eligibility, submit a request with{' '}
+          <a target="_blank" href="/contact-us">
+            {' '}
+            Ask VA.{' '}
+          </a>{' '}
+          If you have technical difficulties using this online application, call
+          our MyVA411 main information line at{' '}
+          <va-telephone contact="8006982411" extension="711" />. We're here
```

Comment:
Hey, I'm pretty sure that this doesn't work the way you're expecting. 711 will be dialed after the initial phone number.